src/table/cells/selections.jsx

```diff
@@ -15,10 +16,10 @@ const useStyles = makeStyles({
 
 export default function withSelections(CellComponent) {
   const HOC = (props) => {
-    const { cell, selections } = props;
+    const { cell, selState, selDispatch } = props;
     const classes = useStyles();
-    const handleMouseUp = () => cell.isDim && selections.selectCell(cell);
-    const selectionClass = selections.getSelectionClass(cell);
+    const handleMouseUp = () => cell.isDim && selectCell(cell, selState, selDispatch);
+    const selectionClass = getSelectionClass(cell, selState);
```

Comment:
Before these functions where stored on the `selections` state and we only needed to pass `cell`. Tried this, but then it was using a stale state, so decided to import them directly from selection-utils.